business/services.go

```diff
@@ -184,6 +198,43 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 		drUpdate, drDelete = getUpdateDeletePermissions(in.k8s, namespace, DestinationRules, "")
 	}()
 
+	var eTraces int
+	go func() {
+		// Maybe a future jaeger business layer
+		defer wg.Done()
+		if config.Get().ExternalServices.Jaeger.Service != "" {
+			u, err := url.Parse(fmt.Sprintf("http://%s:%d/api/traces", config.Get().ExternalServices.Jaeger.Service, config.Get().ExternalServices.Jaeger.ServicePort))
+			if err != nil {
+				errChan <- err
+			} else {
+				q := u.Query()
+				q.Set("lookback", "1h")
+				q.Set("service", service)
+				t := time.Now().UnixNano() / 1000
+				q.Set("start", fmt.Sprintf("%d", t-60*60*1000*1000))
```

Comment:
So, it queries a fixed interval of 60m, right ? should be this be read from a parameter instead ?

Reply:
mmm I don't know....the idea was get the error traces in the last 60 minutes, we can set this by default and let to pass this like queryparam, I show in the UI up to 100 error Traces maybe we should change this too